native/src/routes/SearchModal.tsx

```diff
@@ -44,7 +44,7 @@ const SearchModal = ({
   const theme = useContext(ThemeContext)
   const { t } = useTranslation('search')
 
-  const searchResults = useSearch(allPossibleResults, query, 'async')
+  const searchResults = useSearch(allPossibleResults, query, true)
```

Comment:
🔧  I specifically decided against using a random boolean here because of Steffi's clean code talk :)

Reply:
yes okay but then i have to check here again a string and set a boolean

Reply:
Then I think something like this would be better:
``` ts
const useSearch = (allPossibleResults: SearchResult[], query: string, mode: 'async' | 'concurrent'): SearchResult[] => {
```
or even
``` ts
const useSearch = (allPossibleResults: SearchResult[], query: string, options: { async: boolean }): SearchResult[] => {
```

Reply:
refactored function